resample/bootstrap.py

```diff
@@ -141,11 +137,23 @@ def confidence_interval(
     cl : float, default : 0.95
         Confidence level. Asymptotically, this is the probability that the interval
         contains the true value.
-    ci_method : str, {'percentile', 'student', 'bca'}, optional
-        Confidence interval method. Default is 'percentile'.
+    ci_method : str, {'bca', 'percentile'}, optional
+        Confidence interval method. Default is 'bca'. See notes for details.
     **kwargs
         Keyword arguments forwarded to :func:`resample`.
 
+    Notes
```

Comment:
Should we also include some theoretical notes about when bootstrap confidence intervals will fail (something like the distribution maximum comes to mind) as a warning to users?

Reply:
Yes, good idea. Do you want to add something to this PR or in a separate PR?

Reply:
I think separate PR is fine